recipe/build.sh

```diff
@@ -9,6 +9,13 @@ export PARQUET_HOME=$PREFIX
 export SETUPTOOLS_SCM_PRETEND_VERSION=$PKG_VERSION
 
 cd python
+
 $PYTHON setup.py \
-        build_ext --build-type=release --with-parquet --with-plasma --with-orc \
-        install --single-version-externally-managed --record=record.txt
+        build_ext --build-type=release \
+                  --with-flight \
+                  --with-orc \
+                  --with-plasma \
+                  --with-parquet \
+                  --with-gandiva \
```

Comment:
Let's use environment variables here as well. Passing options on the `setup.py` command-line is a bit cumbersome and error-prone.